schemas/webpack.d.ts

```diff
@@ -0,0 +1,364 @@
+
+        // Similar to `output.devtoolModuleFilenameTemplate`, but used in the case of duplicate module identifiers.
+        devtoolFallbackModuleFilenameTemplate?: string | Function;
+
+        // Enable line to line mapped mode for all/specified modules. Line to line mapped mode uses a simple SourceMap where each line of the generated source is mapped to the same line of the original source. It’s a performance optimization. Only use it if your performance need to be better and you are sure that input lines match which generated lines.
+        // Boolean: `true` enables it for all modules (not recommended)
+        // Object: An object similar to `module.loaders` enables it for specific files.
+        devtoolLineToLine?: boolean | OutputDevtoolLineToLine;
+
+        // Filename template string of function for the sources array in a generated SourceMap.
+        devtoolModuleFilenameTemplate?: string | Function;
+
+        // Specifies the name of each output file on disk. You must **not** specify an absolute path here! The `output.path` option determines the location on disk the files are written to, filename is used solely for naming the individual files.
+        filename?: string;
+        hashDigest?: string;
+        hashDigestLength?: number;
+        hashFunction?: string;
+
+        // The filename of the Hot Update Chunks. They are inside the output.path directory.
+        hotUpdateChunkFilename?: string;
+
+        // The JSONP function used by webpack for async loading of hot update chunks.
+        hotUpdateFunction?: string;
+
+        // The filename of the Hot Update Main File. It is inside the `output.path` directory.
+        hotUpdateMainFilename?: string;
+
+        // The JSONP function used by webpack for async loading of chunks.
+        jsonpFunction?: string;
+
+        // If set, export the bundle as library. `output.library` is the name.
+        library?: string | string[];
+
+        libraryTarget?: OutputLibraryTarget;
+
+        // The output directory as **absolute path** (required).
+        path?: string;
+
+        // Include comments with information about the modules.
+        pathinfo?: boolean;
+
+        // The `publicPath` specifies the public URL address of the output files when referenced in a browser.
+        publicPath?: string;
+
+        // The filename of the SourceMaps for the JavaScript files. They are inside the `output.path` directory.
+        sourceMapFilename?: string;
+
+        // Prefixes every line of the source in the bundle with this string.
+        sourcePrefix?: string;
+
+        // If `output.libraryTarget` is set to umd and `output.library` is set, setting this to true will name the AMD module.
+        umdNamedDefine?: boolean;
+    }
+
+    interface OutputAuxiliaryComment {
+        amd?: string;
+        commonjs?: string;
+        commonjs2?: string;
+        root?: string;
+    }
+
+    interface OutputDevtoolLineToLine {
```

Comment:
They both have `include`, `exclude` and `test` keys, but they don't necessarily seem to work the same. Would be nice if both the implementation and the object format were the same.